src/plugins/intel_npu/src/backend/include/zero_backend.hpp

```diff
@@ -29,6 +29,14 @@ class ZeroEngineBackend final : public IEngineBackend {
     bool isWorkloadTypeSupported() const override;
 
     void* getContext() const override;
+    void updateInfo(const Config& config) override {
```

Comment:
Why not put this implementation in `intel_npu/src/backend/src/zero_backend.cpp` instead? Or make it inline

Reply:
`ZeroEngineBackend::updateInfo` inherits from [IEngineBackend::updateInfo()](https://github.com/DanLiu2Intel/openvino/blob/fix_NPUplugin_log_usage/src/plugins/intel_npu/src/al/include/npu.hpp#L47) (added by this pr) and `updateInfo()` needs to be overloaded in both zerobackend and imdbackend to update their respective log levels. Will move this part implementation  to the `zero_backend.cpp` file.